docs/cogs/information/bot_info.md

```diff
@@ -1 +1,57 @@
 # Bot Info
+
+This cog contains information about the bot and its tasks.
+
+## Commands
+
+### `info`
+The `.info` command shows information about the bot.
+```css
+.[info|infos|about]
+```
+
+The information given by this command include:
```

Comment:
```suggestion
The information given by this command includes:
```
Since information is uncountable, it is grammatically singular. Regardless of the meaning, it should be "includes".